packages/terra-clinical-item-display/src/ItemDisplay.jsx

```diff
@@ -43,6 +44,16 @@ const propTypes = {
    * One of `'center'`, `'top'`, `'inline'`.
    */
   iconAlignment: PropTypes.oneOf(['center', 'top', 'inline']),
+  /**
```

Comment:
I can also add the "Accessibility" badge here, that will show in the doc site props